core/src/test/java/com/linecorp/armeria/server/management/ManagementServiceTest.java

```diff
@@ -84,7 +85,9 @@ void threadDumpWithJson() throws Exception {
 
     @Test
     void heapDump() throws InterruptedException {
-        final WebClient client = WebClient.of(server.httpUri());
+        final WebClient client = WebClient.builder(server.httpUri())
+                                          .responseTimeout(Duration.ofSeconds(45)) // Heap dump can take time.
```

Comment:
It seems like the request timeout of the server is not increased?

Reply:
Increased to 50 seconds